config/deploy/kustomization.yaml

```diff
@@ -1,5 +1,9 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
 namespace: kuadrant-system
-
 resources:
   - ../default
   - ../dependencies
+patchesStrategicMerge:
+- manager_debug_mode.yaml
```

Comment:
Are we defaulting to `debug` on `make deploy` now?

Reply:
yes! we can no longer run `make run` to run locally (an actual process in your local machine) which was configured with debug level to see all details. 

So, now, we (developers) have only "make deploy" dev/Testing deployment option so I decided to deploy with debug to have all available logs. 

This is not affecting OLM deployments for which `LOG_LEVEL` defaults to "INFO" level `LOG_MODE` to `production`

Reply:
Hmmm. IDK. I think debug is OK maybe for `local-deploy`, but `deploy` is a legit target that blindly applies the deploy manifests to the kube context. It seems risky to enable debug by default. If this eventually ends up propagated to internal components config such as auth, it would leak sensitive data to the logs by default.

Maybe commenting this patch and let devs decide when to enable it?

Reply:
Envoy is configured to download some file from the operator. If envoy runs in kubernetes and the operator in your local machine.... what can we do to make that download happen?

I was also heavily using `make run`. But the decision to serve the binary from the operator makes it very hard to keep that way.

Reply:
I am happy to keep if you want.. but does not work for all the use cases.

Reply:
> Hmmm. IDK. I think debug is OK maybe for local-deploy, but deploy is a legit target that blindly applies the deploy manifests to the kubectl context. It seems risky to enable debug by default. If this eventually ends up propagated to internal components config such as auth, it would leak sensitive data to the logs by default.